### PR TITLE
Fix error of undeploy.sh

### PR DESCRIPTION
**Pull request by hougangliu**
`kubectl delete resourceType namespace/resourceName` is invalid.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/687)

### CODE REVIEW

File: examples/v1alpha2/hyperband-example.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ metadata:
   namespace: kubeflow
   name: hyperband-example
 spec:
+  parallelTrialCount: 9
```
Comment (johnugeorge) [inline examples/v1alpha2/hyperband-example.yaml]: may be added maxTrials too?

Comment (hougangliu) [reply examples/v1alpha2/hyperband-example.yaml]: hyperband algorithm can terminate itself when loop exceeds. we doesn't need set maxTrials for it.

**Pull request merged**